documentation/jetty-documentation/src/main/asciidoc/contribution-guide/3-source.adoc

```diff
@@ -26,12 +26,13 @@ The https://github.com/eclipse/jetty.project[Eclipse Jetty project] is located a
 .Active Eclipse Jetty Branches
 [cols="4"]
 |===
-| https://github.com/eclipse/jetty.project/tree/jetty-10.0.x[jetty-11.0.x] | Development | Servlet 5.0 | Java 11+
+| https://github.com/eclipse/jetty.project/tree/jetty-12.0.x[jetty-12.0.x] | Development | Servlet 6.0 | Java 17+
+| https://github.com/eclipse/jetty.project/tree/jetty-11.0.x[jetty-11.0.x] | Development | Servlet 5.0 | Java 11+
 | https://github.com/eclipse/jetty.project/tree/jetty-10.0.x[jetty-10.0.x] | Development (default branch) | Servlet 4.0 | Java 11+
-| https://github.com/eclipse/jetty.project/tree/jetty-9.4.x[jetty-9.4.x] | Maintenance | Servlet 3.1 | Java 8
-| https://github.com/eclipse/jetty.project/tree/jetty-9.3.x[jetty-9.3.x] | Maintenance | Servlet 3.0 | Java 8
-| https://github.com/eclipse/jetty.project/tree/jetty-8.1.x[jetty-8.1.x] | Historical | Servlet 2.0 | Java 7
-| https://github.com/eclipse/jetty.project/tree/jetty-7[jetty-7] | Mythical | Servlet 1.0 | Java 6
+| https://github.com/eclipse/jetty.project/tree/jetty-9.4.x[jetty-9.4.x] | End of Community Support | Servlet 3.1 | Java 8
+| https://github.com/eclipse/jetty.project/tree/jetty-9.3.x[jetty-9.3.x] | End of Life | Servlet 3.1 | Java 8
+| jetty-8 | Historical | Servlet 3.1 | Java 6
+| jetty-7 | Mythical | Servlet 2.5 | Java 5
```

Comment:
The title of this table is "Active Eclipse Jetty Branches".

I don't think jetty-8 and jetty-7 make the grade for this table.

Reply:
And even `jetty-9.3.x` isn't considered "Active" anymore.